src/utils/sortObjectKeys.ts

```diff
@@ -0,0 +1,44 @@
+import { Settings, SortOrder } from "../types/settings.d";
+import { GenericObject } from "../types/shared";
+
+const sortHandlers: Record<
+  SortOrder,
+  (object: GenericObject) => GenericObject
+> = {
+  [SortOrder.ASC]: (object: GenericObject) =>
+    Object.fromEntries(
+      Object.entries(object).sort(([a], [b]) => a.localeCompare(b))
+    ),
+  [SortOrder.DESC]: (object: GenericObject) =>
+    Object.fromEntries(
+      Object.entries(object).sort(([a], [b]) => b.localeCompare(a))
+    ),
+  [SortOrder.NONE]: (object: GenericObject) => object,
+};
+
+function sortObjectKeys(obj: any, sort: SortOrder = SortOrder.NONE): any {
+  const sortHandler = sortHandlers[sort];
+
+  if (!sortHandler) {
+    throw new Error(
+      `The sort option you've provided is not supported, please try one of the next: ${Object.values(SortOrder).join(", ")}`
+    );
+  }
+
+  if (sort === SortOrder.NONE || typeof obj !== "object" || obj === null) {
+    return obj;
+  }
```

Comment:
```suggestion
  if (sort === SortOrder.NONE || typeof obj !== "object" || obj === null) {
    return obj;
  }

  const sortHandler = sortHandlers[sort];

  if (!sortHandler) {
    throw new Error(
      `The sort option you've provided is not supported, please try one of the next: ${Object.values(SortOrder).join(", ")}`
    );
  }
```

Reply:
Moving that validation at top you have covered the case where sortHandler is none, so you can avoid creating unneeded handler for SortOrder.NONE.

What do you think?